app/views/notebook_mailer/feedback.html.slim

```diff
@@ -11,9 +11,10 @@ p
     ==@feedback.general_feedback
     | "
 br
-- if @feedback.worked == false
-  p
-    | Additionally, the user indicated that the notebook was run and it was not successful.
+-if @feedback.worked == true
```

Comment:
You need to check if ran is true as well.
Workflow: Check that you ran it
Check that it didn't work
add feedback for how it didn't work
check that you didn't run it
submit feedback
Email says they reported it didn't work, even though that option was no longer visible on the form.